core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -408,13 +413,17 @@ export class PickerColumnInternal implements ComponentInterface {
   };
 
   get activeItem() {
-    return getElementRoot(this.el).querySelector(
-      `.picker-item[data-value="${this.value}"]:not([disabled])`
-    ) as HTMLElement | null;
+    // If the whole picker column is disabled, the current value should appear active
+    // If the current value item is specifically disabled, it should not appear active
+    const selector = `.picker-item[data-value="${this.value}"]:not(${
```

Comment:
Why did this change? `.picker-item-empty` will never have the `data-value` attribute to begin with, so this selector will never match a `.picker-item-empty` element. Additionally, we disable the `.picker-item` button if the component's `disabled` prop is `true`. As a result, the original selector should still be sufficient.

Reply:
I can remove the `.picker-item-empty` part. But the behavior does need to be different for a disabled column vs disabled items (as far as I understand.) The behavior previously was that it should not scroll to a disabled item. But when the whole column is disabled, it *should* scroll to a disabled item. e.g. if the default value in a calendar picker is today, it should scroll to that even if the column is disabled. You can see a native example of that in my text in the PR description.

Reply:
Ok makes sense! Does the selector need to be `.picker-item-disabled` or are we able to keep using `[disabled]`?

Reply:
Oh, true, we can go back to `[disabled]` after the refactoring I did previously.